tony-portal/app/views/tableMetadata.scala.html

```diff
@@ -1,12 +1,31 @@
 @(metricsList: Collection[JobMetadata])
 
-<html> <body>
+<html>
+
+  <head>
+
     <!-- Required meta tags for Bootstrap-->
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <!-- Bootstrap CSS -->
-  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css" integrity="sha384-GJzZqFGwb1QTTN6wy59ffF1BuGJpLSa9DkKMp0DgiMDm4iYMj70gZWKYbI706tWS" crossorigin="anonymous">
+  <meta charset="utf-8"/>
```

Comment:
Nit: I think this should be indented two more spaces

Reply:
@erwa reformatted code, please take a look